src/wp-content/themes/twentytwentyone/assets/sass/04-elements/blockquote.scss

```diff
@@ -17,19 +23,18 @@ blockquote {
 	}
 
 	p {
-		letter-spacing: var(--heading--letter-spacing-h4);
-		font-family: var(--quote--font-family);
-		font-size: var(--quote--font-size);
-		font-style: var(--quote--font-style);
-		font-weight: var(--quote--font-weight);
-		line-height: var(--quote--line-height);
+		letter-spacing: inherit;
+		font-family: inherit;
+		font-size: inherit;
+		font-style: inherit;
+		font-weight: inherit;
+		line-height: inherit;
 	}
 
 	cite,
 	footer {
 		font-weight: normal;
 		color: var(--global--color-primary);
-		font-size: var(--global--font-size-xs);
```

Comment:
I removed the cite font size to make it change relatively to the quote block font size.